book/website/project-design/sdp/pd-sdp-private-learning.md

```diff
@@ -0,0 +1,51 @@
+(pd-sdp-private-learning)=
+
+# Privacy-preserving Machine learning
+
+Much current research in data science involves machine learning (ML) models interacting with data sourced from a large number of individuals with significant variation in the general level of awareness, consent, and understanding of research goals. As such, researchers have a responsibility to protect the confidentiality and privacy of the people whose data is being processed. At the same time, sharing both data and trained models drives scientific advancement and promotes important social goals in open and transparent science.
+
+It's important to note that local and international regulations such as the [General Data Protection Regulation](https://gdpr-info.eu/) (GDPR) and the EU’s policy on [trustworthy AI](https://digital-strategy.ec.europa.eu/en/policies/european-approach-artificial-intelligence) also establish legal duties and principles on privacy protection that the following tools may help researchers meet.
```

Comment:
I don't fully understand this sentence - is there a word missing?

Reply:
I'm sorry, which sentence are you referring to? I don't have trouble understanding it—probably just me skipping over the problem because I wrote it.

Reply:
aaah actually, I just read it again and now it works (it was the 2nd sentence) - yesterday my brain somehow did not get  the back reference from "tools may help researchers meet' -> to establish legal duties and principles. My bad, it's fine!